marketplace/gcp/tidb-operator-enterprise/chart/tidb-mp/values.yaml

```diff
@@ -0,0 +1,51 @@
+tidb-operator:
+  rbac:
+    create: false
+  clusterScoped: false
+  controllerManager:
+    serviceAccount:
+  operatorImage:
+
+
+tidb-cluster:
+  # Assume one cluster per namespace
+  clusterName: db
+  rbac:
+    create: false
+  
+  fullbackup:
+    # Need to get the backup images pushed to GCR first.
+    create: false
+    password:
+
+  monitor:
```

Comment:
How does this works if not running as a sidecar?

Reply:
It is running as a sidecar in just the monitor pod. It just counts its own time right now, its not quite what we want yet.

Reply:
Maybe we should consider a general approach to run sidecar containers in tidb-operator @weekface

Reply:
The problem with the `toYaml extraContainers` approach that I originally tried is that the extraContainers cannot be helm templated. However, that may work fine for other sidecar use cases.